src/components/SuggestionsList/SuggestionsListItem.tsx

```diff
@@ -0,0 +1,82 @@
+/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
+import '../../yamui';
+import * as React from 'react';
+import { NestableBaseComponentProps } from '../../util/BaseComponent/props';
+import Clickable from '../Clickable';
+import MediaObject, { MediaObjectSize } from '../MediaObject';
+import Avatar, { AvatarSize } from '../Avatar';
+import Block, { GutterSize } from '../Block';
+import { FixedGridRow, FixedGridColumn } from '../FixedGrid';
+import './SuggestionsListItem.css';
+
+export interface SuggestionItem {
+  id: string | number;
+  imageUrl?: string;
+  name: string;
+  description?: string;
+}
+
+export interface SuggestionsListItemProps extends SuggestionItem, NestableBaseComponentProps {
+  search: string;
+  isHovered: boolean;
+  isSelected: boolean;
+  onHover(id: string | number): void;
+  onSelect(item: SuggestionItem): void;
+}
+
+const selectedClass = 'y-suggestions-list-item--selected';
+const hoveredClass = 'y-suggestions-list-item--hover';
+const highlightedClass = 'y-suggestions-list-item--highlight';
+
+const getHighlightedName = (name: string, search: string) => {
+  return name.split(new RegExp(`(${search})`, 'gi')).map((item: string, index: number) => {
+    const className = search.toLowerCase() === item.toLowerCase() ? highlightedClass : undefined;
+    return <span key={index} className={className}>{item}</span>;
+  });
+};
+
+export default class SuggestionsListItem extends React.PureComponent<SuggestionsListItemProps, {}> {
+  render() {
+    const { isSelected, isHovered, name, search, imageUrl, description } = this.props;
+    const avatar = (
+      <Avatar
+        imageUrl={imageUrl}
+        name={name}
+        size={AvatarSize.SMALL}
+        initials="" // !!
+      />);
+    const horizontalSpacer = <FixedGridColumn width={0} fixed={true} />;
+    const className = isHovered ? hoveredClass : isSelected ? selectedClass : undefined;
+    const title = getHighlightedName(name, search);
+    return (
+      <Clickable
+        onClick={this.onClick}
+        unstyled={true}
+        block={true}
+        onHover={this.onHover}
+        className={className}
+      >
+        <Block padding={GutterSize.SMALL} >
+          <FixedGridRow gutterSize={GutterSize.SMALL}>
+            {horizontalSpacer}
+            <FixedGridColumn>
+              <MediaObject
+                size={MediaObjectSize.SMALL}
+                imageContent={avatar}
+                titleContent={title}
+                metadataContent={description}
+              />
+            </FixedGridColumn>
+            {horizontalSpacer}
+          </FixedGridRow>
+        </Block>
+      </Clickable>);
```

Comment:
@swese44 this feels like a ton of components for something quite simple, no?

Reply:
i'd kill the fixed grid and add an extra 4px horizontal margin on the MediaObject. i will separately ask Design if these can have consistent padding instead